projects/components/src/select/select.component.ts

```diff
@@ -90,6 +99,18 @@ import { SelectSize } from './select-size';
               </ht-icon>
             </div>
           </div>
+
+          <ng-template #controlItemTemplate let-items="controlItems">
```

Comment:
any reason to have the separate template rather than doing this inline? It's not reused or passed around, and it seems like it makes it a bit less readable. On the other hand, I could see using a template for the item and sharing it in both the control and standard branches.

Reply:
The reason for using a template is so that we can extend this to a `bottom` control group usecase later.
Reason to not combine the options and control items is because of the selected status icon. Control items do not need the selected status, but options do.

Reply:
Ah right. I think we'd benefit from sharing the template though given the amount of duplication. For example, the control item here will render item.icon, but it doesn't look like the regular items render it. Could we use, in the same pattern you have, something like `let-show-selection-status`?